src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/CreateUserSpec.scala

```diff
@@ -277,6 +309,20 @@ class CreateUserSpec extends UserServiceTestTraits {
 
   describe("An invited User") {
 
+    it("should be able to be invited with no marketing consent") {
+      // Arrange
+      val invitedGoogleUser = genWorkbenchUserGoogle.sample.get
+      val directoryDAO = MockDirectoryDaoBuilder(allUsersGroup).build
+      val cloudExtensions = MockCloudExtensionsBuilder(allUsersGroup).build
+      val userService = new UserService(directoryDAO, cloudExtensions, Seq.empty, defaultTosService)
+
+      // Act
+      val invitedUserStatus = runAndWait(userService.inviteUser(invitedGoogleUser.email, samRequestContext))
+
+      // Assert
+      verify(directoryDAO).setUserAttributes(SamUserAttributes(invitedUserStatus.userSubjectId, marketingConsent = false), samRequestContext)
+    }
+
```

Comment:
Add a test where a user tries to grab another user's attributes?

Reply:
There's no api endpoint to specify a user id of the user to get attributes of, so that's not possible.

Reply:
Oh right duh